fineract-provider/src/main/java/org/apache/fineract/infrastructure/report/service/PentahoReportingProcessServiceImpl.java

```diff
@@ -86,7 +89,10 @@ public Response processRequest(final String reportName, final MultivaluedMap<Str
             throw new PlatformDataIntegrityException("error.msg.invalid.outputType", "No matching Output Type: " + outputType);
         }
 
-        final var reportPath = MIFOS_BASE_DIR + File.separator + "pentahoReports" + File.separator + reportName + ".prpt";
+        var reportPath = MIFOS_BASE_DIR + File.separator + "pentahoReports" + File.separator + reportName + ".prpt";
+        if (isPentahoReportEngineRunOnPostgres) {
+            reportPath = MIFOS_BASE_DIR + File.separator + "pentahoReportsPostgres" + File.separator + reportName + ".prpt";
+        }
         var outPutInfo = "Report path: " + reportPath;
```

Comment:
You can add a new property `pentahoFolderName` to `application.properties` and use it here instead of doing this so you would just have 1 line instead of 4:

`var reportPath = MIFOS_BASE_DIR + File.separator + fineractProperties.getPentahoFolderName() + File.separator + reportName + ".prpt";`

Reply:
@hagafrank  This has been fixed